certified-connectors/Kapish Power Connect/apiDefinition.swagger.json

```diff
@@ -0,0 +1,8957 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Micro Focus Content Manager",
```

Comment:
@BrettHankinson can you please make sure that the folder name is similar to the connector title. this will make it easier to find the connector in our files.